service/informer/informer.go

```diff
@@ -148,39 +148,37 @@ func (s *Service) bootWithError() error {
 
 	// If the TPO was not found the project list is empty, which means we
 	// initialize it.
-	if len(TPO.Spec.Projects) == 0 {
```

Comment:
When testing on lycan I noticed partially up to date TPOs do not get updated initially because we only did it in case the list is empty. Now we fetch initially anyway.